File: app/src/main/java/com/libmailcore/androidexample/MessageViewListActivity.java
package com.libmailcore.androidexample;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;

import com.libmailcore.IMAPMessage;


public class MessageViewListActivity extends Activity implements MessageViewListFragment.Callbacks {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_messageview_list);
    }


    @Override
    public void onItemSelected(IMAPMessage msg) {
        MessagesSyncManager.singleton().currentMessage = msg;
        Intent detailIntent = new Intent(this, MessageViewDetailActivity.class);
        startActivity(detailIntent);

    }
}

File: app/src/main/java/com/kongpf8848/dmail/messagelist/MessageViewListActivity.kt
package com.kongpf8848.dmail.messagelist

import android.content.Intent
import android.os.Bundle
import com.kongpf8848.dmail.R
import com.kongpf8848.dmail.mailcore.MailCore2Api
import com.libmailcore.IMAPMessage
import okhttp3.OkHttpClient

class MessageViewListActivity : com.kongpf8848.dmail.library.base.BaseActivity(), MessageViewListFragment.Callbacks {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_messageview_list)
    }

    override fun onItemSelected(msg: IMAPMessage) {
        MailCore2Api.instance.currentMessage = msg
        val detailIntent = Intent(this, MessageViewDetailActivity::class.java)
        startActivity(detailIntent)

        var cc=OkHttpClient();
        val request=OkHttpClient


    }
}